plugin/src/main/groovy/com/novoda/gradle/command/AndroidCommandPluginExtension.groovy

```diff
@@ -83,6 +86,14 @@ public class AndroidCommandPluginExtension {
         monkey
     }
 
+    void script(Action<NamedDomainObjectContainer<InputSpec>> action) {
+        action.execute(scripts)
```

Comment:
You can postpone the creation of the task waiting for the end of the configuration phase, after which the `scripts` container will be filled with all the entries. we do something similar in the other plugin :)